components/script/dom/serviceworkerglobalscope.rs

```diff
@@ -315,7 +315,7 @@ impl ServiceWorkerGlobalScope {
                     },
                 };
 
-                let runtime = new_rt_and_cx();
+                let runtime = new_rt_and_cx(None);
```

Comment:
I think you could make the `own_sender` above into a `ScriptChan` and use it here as well.